DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -683,7 +675,7 @@ public void tSetupItemList_Drums()
                 "判定ラインが上になり、\n" +
                 "ノーツが下から上へ\n"+
                 "流れます。",
-                "The scroll way is reversed. Drums chips flow from the bottom to the top.");
+				"Drum chips scroll up the lanes instead of down the lanes.");
```

Comment:
```suggestion
		"Drum chips scroll up the lanes instead of down the lanes.");
```
Sorry for the indentation here.